spec/features/admin/admin_updated_orders_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+feature 'Logged in admin' do
+  let!(:category) { Category.create!(name: "Monster") }
+
+  scenario 'can update orders' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.